mockall/src/lib.rs

```diff
@@ -1273,7 +1273,7 @@ pub mod examples;
 /// # Limitations
 ///
 /// `#[automock]` can't handle everything.  There are some cases where
-/// you will need to use [`mock`] instead:
+/// you will need to use [`mock!`] instead:
```

Comment:
The link works as-is.  Does it still work after this change?  Did you generate the docs to check?

Reply:
> The link works as-is. Does it still work after this change? Did you generate the docs to check?

Indeed I've checked them ;)